cli/cmd/inject_test.go

```diff
@@ -10,6 +10,7 @@ import (
 )
 
 func TestInjectYAML(t *testing.T) {
+	conduitVersion = "latest" // override "undefined"
```

Comment:
Two comments here:
1) TIOLI but I like using some obvious test/fake strings as test data to make it easier to debug errors. In this case I'd probably use something like `"this-is-some-version"` or something like that that is easily grep'able.
2) I am quite nervous about overriding this quasi-global variable. Could we change this instead so that `InjectYAML` takes in the version (and any other data currently read from parameters)? It would also fit better how other commands work, IIRC

Reply:
Good call -- have made this changes in a follow-up commit.